DuckDuckGoTests/AutofillPixelReporterTests.swift

```diff
@@ -0,0 +1,193 @@
+//
+//  AutofillPixelReporterTests.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+@testable import DuckDuckGo
+@testable import Core
+@testable import BrowserServicesKit
+
+final class AutofillPixelReporterTests: XCTestCase {
+
+    var statisticsStorage: MockStatisticsStore!
+    private let vault = (try? MockSecureVaultFactory.makeVault(reporter: nil))!
+    var autofillPixelReporter: AutofillPixelReporter!
```

Comment:
Could make these private for consistency if you preferred:

```suggestion
    private var statisticsStorage: MockStatisticsStore!
    private let vault = (try? MockSecureVaultFactory.makeVault(reporter: nil))!
    private  var autofillPixelReporter: AutofillPixelReporter!
```